tcf_website/templates/reviews/review.html

```diff
@@ -36,48 +40,69 @@
-                            {{ review.instructor_rating }}
-                        </div>
-                        <div class="ml-4 text-muted">
-                            <i class="far fa-smile-beam"></i>
-                            {{ review.enjoyability }}
-                        </div>
-                        <div class="ml-4 text-muted">
-                            <i class="fa fa-dumbbell fa-fw"></i>
-                            {{ review.difficulty }}
-                        </div>
-                        <div class="ml-4 text-muted">
-                            <i class="fa fa-hourglass-half fa-fw"></i>
-                            {{ review.hours_per_week }}
+                <div class="review-content">
+                  <p class="card-text mb-2" style="white-space: pre-line;">{{ review.text }}</p>
+                </div>
+
+                <div class="review-footer row mt-3">
+                    <!-- Review Ratings -->
+                    <div class="review-ratings col-sm-8 d-flex align-items-end">
+                        <div class="d-flex text-center flex-wrap mb-2">
+                            <div id="review-average" class="font-weight-bold">
+                                <i class="fa fa-star fa-fw"></i>
+                                {{ review.average|floatformat:2 }}
+                            </div>
+                            <div class="ml-4 text-muted">
+                                <i class="fas fa-thumbs-up"></i>
+                                {{ review.recommendability }}
+                            </div>
+                            <div class="ml-4 text-muted">
+                                <i class="fa fa-user"></i>
+                                {{ review.instructor_rating }}
+                            </div>
+                            <div class="ml-4 text-muted">
+                                <i class="far fa-smile-beam"></i>
+                                {{ review.enjoyability }}
+                            </div>
+                            <div class="ml-4 text-muted">
+                                <i class="fa fa-dumbbell fa-fw"></i>
+                                {{ review.difficulty }}
+                            </div>
+                            <div class="ml-4 text-muted">
+                                <i class="fa fa-hourglass-half fa-fw"></i>
+                                {{ review.hours_per_week }}
+                            </div>
                         </div>
                     </div>
 
-                    <div class="text-right">
-                        <p>{{ review.semester }}</p>
-                        <p id="date">{{ review.created|date }}</p>
+                    <div class="review-date col-sm-4 pl-0 text-right text-tcf-orange">
+                        <h6 class="font-weight-bold">{{ review.semester }}</h6>
+                        <h6 id="date">Updated {{ review.created|date }}</h6>
```

Comment:
Could you make this a different color (black, grey or muted)? Since it's less important information

Reply:
I think it's already "muted" since the semester date above it is bold?
I feel it's pretty important info tho

Reply:
eh I'll make the change to muted